### PR TITLE
update jsonschema of join task

### PR DESCRIPTION
**Pull request by gdraynz**

### CODE REVIEW

File: nyuki/workflow/tasks/__init__.py [hunk 1]
```diff
@@ -12,19 +12,22 @@
...
-            'maxItems': 64,
-            'uniqueItems': True,
-            'items': {
-                'type': 'string',
-                'maxLength': 1024
-            }
+            'anyOf': [
+                {
+                    'type': 'array',
+                    'minItems': 2,
```
Comment (lerignoux) [inline nyuki/workflow/tasks/__init__.py]: why not min 1 ? we could wait for only one parent

File: nyuki/workflow/tasks/__init__.py [hunk 2]
```diff
@@ -12,19 +12,22 @@
...
+                    'maxItems': 64,
+                    'uniqueItems': True,
+                    'items': {
+                        'type': 'string',
```
Comment (lerignoux) [inline nyuki/workflow/tasks/__init__.py]: shouldn't items be a ref to TASKID_SCHEMA ?

File: nyuki/workflow/tasks/__init__.py (hunk 1 above)
Comment (gdraynz) [inline nyuki/workflow/tasks/__init__.py]: If we have to wait for only one task, there's no point having this task at all.
This is a join, used to gather results of multiple tasks.

File: nyuki/workflow/tasks/__init__.py (hunk 2 above)
Comment (gdraynz) [inline nyuki/workflow/tasks/__init__.py]: Right

**Pull request merged**